.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: perl
+perl:
+    - "5.18"
+    - "5.16"
+    - "5.14"
+    - "5.12"
+services:
+    - redis-server
+before_install:
+    - "cpanm JSON"
+    - "cpanm Date::Utility"
+    - "cpanm Format::Util::Numbers"
+    - "cpanm Test::FailWarnings"
+    - "cpanm Test::Most"
+    - "cpanm Devel::Cover::Report::Coveralls"
```

Comment:
I thought that it is needed for "-report coveralls" in the last line of the file. I will check and If not needed, I will delete it.